IndexedDB/idbfactory-deleteDatabase-opaque-origin.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>IDBFactory.deleteDatabase() and opaque origins</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+
+function load_iframe(src, sandbox) {
+  return new Promise(resolve => {
+    const iframe = document.createElement('iframe');
+    iframe.onload = () => { resolve(iframe); };
+    if (sandbox)
+      iframe.sandbox = sandbox;
+    iframe.srcdoc = src;
+    iframe.style.display = 'none';
+    document.documentElement.appendChild(iframe);
+  });
+}
+
+function wait_for_message() {
+  return new Promise(resolve => {
+    self.addEventListener('message', function listener(e) {
```

Comment:
`promise_test`s run sequentially, so this should not be the case.

https://github.com/w3c/testharness.js/blob/master/testharness.js#L525

https://github.com/w3c/testharness.js/blob/master/docs/api.md

> Unlike Asynchronous Tests, Promise Tests don't start running until after the previous Promise Test finishes.

... but defense in depth is good, so adding that check.